.gitignore

```diff
@@ -1,3 +1,3 @@
-node_modules
+/node_modules
```

Comment:
this is to ignore `licenser/node_modules` and not ignore `test/fixtures/foo/node_modules`